app/services/database.js

```diff
@@ -176,6 +180,7 @@ export default Ember.Service.extend(PouchAdapterUtils, {
   _createLocalDB(localDBName, pouchOptions) {
     return new Ember.RSVP.Promise(function(resolve, reject) {
       let localDB = new PouchDB(localDBName, pouchOptions);
+      window.localMainDB = localDB;
```

Comment:
Like remoteDB, Using a global is unneeded, but we should set the localDB as a property on the database service so that the application adapter has access to it:
```js
this.set('localDb', localDb);
```
Then in the application adapter you can access via:
```js
let localDB = this.get('database').get('localDB');
```